src/constants.js

```diff
@@ -26,3 +26,7 @@ export const APPS_ROUTING_TO_NAME = new Map([
 export const APP_CUSTOM_NAME = new Map([['Tao Voting', 'Delegate Voting']])
 
 export const MAIN_HEADER_HEIGHT = 8 * GU
+
+export const VOTING_REFERENCE_SEPARATOR = '|'
+
+export const VOTING_DESCRIBED_STEP_PREFIX = 'Create a new vote about '
```

Comment:
We can remove this if we use the `isEmptyCallScript` function mentioned above

Reply:
I tried but it does not return true, i.e this is a callscript on the delay for a signaling vote ```"0x000000012c7109166a3c1d306c51aa092445095254dbb99f000000e40a0932da00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000000a30783030303030303031000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000026746869732069732061207469746c657c68747470733a2f2f7777772e676f6f676c652e636f6d0000000000000000000000000000000000000000000000000000"```

Reply:
Oh yeah, you're right. Let's keep the `VOTING_DESCRIBED_STEP_PREFIX` then.

We can close this comment